### PR TITLE
Bug 1833222: Update yaml template meter example to current year

### PR DESCRIPTION
**Pull request by zherman0**
Update yaml template metering example to current year.

Screenshot_2020-05-11 OKD(1)

### CODE REVIEW

File: frontend/public/models/yaml-templates.ts [hunk 1]
```diff
@@ -201,8 +201,8 @@ metadata:
   namespace: openshift-metering
 spec:
   query: namespace-memory-request
-  reportingStart: '2019-01-01T00:00:00Z'
-  reportingEnd: '2019-12-30T23:59:59Z'
+  reportingStart: '2020-01-01T00:00:00Z'
+  reportingEnd: '2020-12-30T23:59:59Z'
```
Comment (spadgett) [inline frontend/public/models/yaml-templates.ts:205]: Does it work to generate these values dynamically from the current date?

```suggestion
  reportingStart: '${new Date().getFullYear()}-01-01T00:00:00Z'
  reportingEnd: '${new Date().getFullYear()}-12-30T23:59:59Z'
```

Comment (spadgett) [reply frontend/public/models/yaml-templates.ts:205]: Is there a reason we don't go to 12-31?

Comment (zherman0) [reply frontend/public/models/yaml-templates.ts]: It does work to generate the values dynamically, but what about job security?  I updated it to 12-31 as well.

Comment (spadgett) [reply frontend/public/models/yaml-templates.ts]: > It does work to generate the values dynamically, but what about job security?

🤔 hm, good point! i didn't consider that

**Pull request merged**